### PR TITLE
Add macOS support

### PR DESCRIPTION
**Pull request by cityofships**
This might come useful from time to time. It assumes working [Brew](https://brew.sh) installation.

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -23,7 +23,9 @@
     name: "{{ os_openstacksdk_package_dependencies }}"
     state: present
   become: True
-  when: os_openstacksdk_install_package_dependencies | bool
+  when:
+    - os_openstacksdk_install_package_dependencies | bool
+    - ansible_facts.os_family != "Darwin"
```
Comment (markgoddard) [inline tasks/main.yml:28]: Seems like we don't need this condition if the list is empty?

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (cityofships) [reply tasks/main.yml:28]: It tries to execute the task without it:
```TASK [stackhpc.os_openstacksdk : Ensure required packages are installed] *********************************************************************************************************************************************************************
task path: /Users/piotr/repos/forks/ansible-role-os-openstacksdk/tasks/main.yml:21
fatal: [localhost]: FAILED! => {
    "changed": false,
    "msg": "Could not find a module for unknown."
}
```
Possibly it's better to put `os_openstacksdk_install_package_dependencies: False` into Darwin.yml.
```suggestion
```

**Comment by cityofships** [comment]
I'll add a package installation part for feature parity

**Review by markgoddard — APPROVED** [review]

**Review by markgoddard — COMMENTED** [review]
Is it actually a draft?

**Comment by cityofships** [comment]
> Is it actually a draft?

It is indeed - disappointing if you're being notified despite this - I was expecting Gerrit-like behaviour.

File: meta/main.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ galaxy_info:
     Role to install the python package openstacksdk
   company: StackHPC Ltd
   license: Apache2
-  min_ansible_version: 2.5
+  min_ansible_version: 2.9
```
Comment (markgoddard) [inline meta/main.yml:9]: Why 2.9?

Comment (cityofships) [reply meta/main.yml:9]: Oldest non-EOL release

Comment (markgoddard) [reply meta/main.yml:9]: It's not really up to this role to dictate that though. Distros may provide support for older releases

**Pull request merged**